README.md

```diff
@@ -45,8 +45,11 @@ Ensure you have [docker](https://docs.docker.com/engine/install/) and [git](http
 1. Create the artifacts needed to run a testnet distributed validator cluster
 
    ```sh
+   # Enter required validator addresses
+   WITHDRAWAL_ADDR=<ENTER YOUR WITHDRAWAL ADDRESS HERE>
```

Comment:
can you please add `FEE_RECIPIENT_ADDR` and `--name` to this section?

Reply:
Added